src/index.html

```diff
@@ -6,7 +6,39 @@
     <title>Calendar</title>
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
-  <body>
-    <h1>Calendar</h1>
+  <body class="page">
+    <div class="calendar calendar--start-day--sun calendar--month-length--29">
+      <div class="calendar__day calendar__day--1"></div>
+      <div class="calendar__day calendar__day--2"></div>
+      <div class="calendar__day calendar__day--3"></div>
+      <div class="calendar__day calendar__day--4"></div>
```

Comment:
Do you really need this modifiers for every day?

Reply:
I use them for this task:
Add a number (Arial 30px) in the center of each day using ::before and [@for]

so I solved it like this:
    @for $index from 1 through 31 {
      &--#{$index} {
        &::before {
          content: '#{$index}';
        }
      }
    }
    
    I don't know better way to do it without modifiers :(

Reply:
And what about `:nth-child`? 🙂